glossary.html

```diff
@@ -20,8 +20,8 @@
         <div class="width-wrap">
             <header id="top">
                 <nav>
-                    <label class="burger" for="burger-toggle"></label>
                     <input type="checkbox" id="burger-toggle">
+                    <label class="burger" for="burger-toggle"></label>
```

Comment:
The label has to come after the checkbox in order for the sibling selector to work